addon/components/table-columns.js

```diff
@@ -8,7 +8,10 @@ const {
 } = Ember;
 
 export default Ember.Component.extend({
-  layout: layout,
+  layout,
+
+  columns: null,
+  tableWidth: Ember.String.htmlSafe('auto'),
```

Comment:
I'm not sure what the best way to handle this case is. By default table columns shouldn't have a width, this is only used for the fixed columns (but they share the same template). Setting this to `auto` here will override the users CSS which seems like something we don't want to do. I can set it to an empty string but then we get `style="width: ;"`, which isn't appealing either. What do you think @cball?